### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by frazarshad**
Changed the README file to better fit this project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,191 +1,31 @@
...
 # with pnpm
-pnpm add --save-dev @synthetixio/synpress
+pnpm add --save-dev @agoric/synpress
 # with npm
-npm install --save-dev @synthetixio/synpress
+npm install --save-dev @agoric/synpress
 # with yarn
-yarn add -D @synthetixio/synpress
+yarn add -D @agoric/synpress
 ```
```
Comment (rabi-siddique) [inline README.md:21]: Wasn't it giving issues with yarn and npm initially?

File: README.md [hunk 2]
```diff
@@ -220,301 +57,49 @@ module.exports = {
...
-      "@testing-library/cypress"
-    ],
-    "outDir": "./output"
-  },
-  "include": ["**/*.*"]
+  ...
+  "scripts": {
+    ...
+    "test:e2e": "EXTENSION=keplr synpress run"
+  }
```
Comment (rabi-siddique) [inline README.md:73]: I think we should elaborate what is `EXTENSION`, a little clarification

File: README.md [hunk 3]
```diff
@@ -220,301 +57,49 @@ module.exports = {
...
-
-launch test runner UI
-
-Options:
-  -cf, --configFile <path>  specify a path to *.js file where configuration values are set
-  -h, --help                display help for command
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+use this config by passing the `--configFile` flag to synpress
+```bash
+EXTENSION=keplr synpress run --configFile=test/e2e/synpress.config.js
```
Comment (rabi-siddique) [inline README.md:93]: Suggestion:

```
{
  ...
  "scripts": {
    ...
    "test:e2e": "EXTENSION=keplr synpress run --configFile=test/e2e/synpress.config.js"
    
  }
}
```

File: README.md [hunk 4]
```diff
@@ -220,301 +57,72 @@ module.exports = {
...
+For an example project, you can take a look at how we've set up tests in this [repository](https://github.com/agoric-labs/synpress/tree/dev/tests/e2e) 
 
 ## 📃 Environmental variables
 
 | Variable                | Description                                                                                                                                                                                                 |
 | ----------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `EXTENSION` (Required)          | Picks which extension to use during tests. `keplr` and `metamask` are the only possible values                                                                                                                              |
 | `SECRET_WORDS`          | Space separated words for the test wallet recovery phrase (mnemonic; 12 words)                                                                                                                              |
```
Comment (rabi-siddique) [inline README.md:115]: I think it should be 1`2/24 words`?

**Review by rabi-siddique — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (frazarshad) [reply README.md]: we actually only support 24 words at the moment. updated it accoridingly

**Pull request merged**